Assignment/solution_student/index.html

```diff
@@ -1,35 +1,46 @@
+<!DOCTYPE>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>About me</title>
+		<link rel="stylesheet" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Lora' rel='stylesheet' type='text/css'>
+		<link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
+	</head>
+	<body>
+		<div class="nav1">
+			<ul>
+				<li><em>About Me</em></li>
+				|
+				<li><a href="resume.html">Portfolio</a></li>
+			</ul>
+		</div>
+		<h1>Joe Johnson</h1>
+		<hr>
+		<div class="img1">
+			<img src="images/avatar.png" alt="Joe Johnson">
+		</div>
+		<h3>I ENJOY LIFE AS A DEVELOPER</h3>
+
+		<p>I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.</p>
+
+		<hr>
+	<div class="nav1">
+		<ul>
+			<li><a href="http://www.facebook.com">Facebook</a></li>
+			|
+			<li><a href="http://www.twitter.com">Twitter</a></li>
+			|
+			<li><a href="http://www.instagram.com">Instagram</a></li>
+			|
+			<li><a href="http://www.linkedin.com">Linked In</a></li>
+
+		</u
```

Comment:
Oops! Looks like you forgot to close your `<ul>` — I can tell in my text editor easily because the syntax highlighting is off: http://d.pr/i/1hIN7